x/registry/types/message_register_top_level_domain_test.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+import (
+	"testing"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/mycel-domain/mycel/testutil/sample"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMsgRegisterTopLevelDomain_ValidateBasic(t *testing.T) {
+	tests := []struct {
+		name string
+		msg  MsgRegisterTopLevelDomain
+		err  error
+	}{
+		{
+			name: "invalid address",
+			msg: MsgRegisterTopLevelDomain{
+				Creator: "invalid_address",
```

Comment:
Add missing argument `name` and `registrationPeriodInYear`

Reply:
added.
also added to existing one `x/registry/types/message_register_second_level_domain_test.go`